sopel/plugin.py

```diff
@@ -245,6 +248,54 @@ def add_attribute(function):
     return add_attribute
 
 
+def rule_lazy(*loaders):
+    """Decorate a callable as a rule with lazy loading.
+
+    :param loaders: one or more functions to generate a list of **compiled**
+                    regexes to match URLs.
```

Comment:
Disagrees with the style we've used in the past.

```suggestion
                    regexes to match URLs
```

Reply:
I *never* remember.